sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/PhpUnitItCoverageResultParser.java

```diff
@@ -30,19 +30,18 @@
 
   /**
    * Instantiates a new php unit coverage result parser.
-   *
-   * @param context    the context
    */
-  public PhpUnitItCoverageResultParser(SensorContext context, FileSystem fileSystem) {
-    super(context, fileSystem);
+  public PhpUnitItCoverageResultParser(FileSystem fileSystem) {
+    super(fileSystem);
     linesToCoverMetric = CoreMetrics.IT_LINES_TO_COVER;
     uncoveredLinesMetric = CoreMetrics.IT_UNCOVERED_LINES;
     coverageLineHitsDataMetric = CoreMetrics.IT_COVERAGE_LINE_HITS_DATA;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "IT_COVERAGE_LINE_HITS_DATA"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)